python/kserve/pyproject.toml

```diff
@@ -35,7 +35,9 @@ exclude = ["kserve/storage/test"]
 [tool.poetry.dependencies]
 python = ">=3.9,<3.13"
 uvicorn = { version = "^0.30.6", extras = ["standard"] }
-fastapi = ">=0.114.1"
+fastapi = ">=0.115.2"
+# pin starlette version to >= 0.40.0 as fastapi supports it
+starlette = ">=0.40.0,<0.41.0"
```

Comment:
why need to pin the starlette version ?

Reply:
We can update the fastapi version which defines this version range for starlette https://fastapi.tiangolo.com/release-notes/#01153

Reply:
great :)

Reply:
@yuzisun @sivanantha321 updated.